Source/WTF/wtf/DateMath.cpp

```diff
@@ -409,18 +410,14 @@ static int findMonth(std::span<const LChar> monthStr)
     if (monthStr.size() < 3)
         return -1;
 
-    std::array<char, 4> needle;
+    std::array<LChar, 3> needle;
     for (unsigned i = 0; i < 3; ++i)
-        needle[i] = static_cast<char>(toASCIILower(monthStr[i]));
-    needle[3] = '\0';
-    const char* haystack = "janfebmaraprmayjunjulaugsepoctnovdec";
-WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
-    const char* str = strstr(haystack, needle.data());
-WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
-    if (str) {
-        int position = static_cast<int>(str - haystack);
-        if (position % 3 == 0)
-            return position / 3;
+        needle[i] = toASCIILower(monthStr[i]);
+    auto haystack = "janfebmaraprmayjunjulaugsepoctnovdec"_span;
+    size_t index = find(haystack, std::span { needle });
+    if (index != notFound) {
```

Comment:
I would have written:
```
    if (size_t index = find(haystack, std::span { needle }); pidIndex != notFound) {
```
I also wonder if there’s some way to get the conversion to `std::span` to happen implicitly in cases like this.